pkg/nsx/services/realizestate/realize_state.go

```diff
@@ -30,21 +39,42 @@ func NewRealizeStateError(msg string) *RealizeStateError {
 	return &RealizeStateError{message: msg}
 }
 
+func IsRealizeStateError(err error) bool {
```

Comment:
not sure why jianwei didn't put  RealizeStateError
in file https://github.com/vmware-tanzu/nsx-operator/blob/main/pkg/nsx/util/errors.go,
should we put all nsx error in this file? @TaoZou1

Reply:
As discussed in gchat, moved those errors and related functions to errors.go, thanks!